docarray/array/document.py

```diff
@@ -388,12 +388,10 @@ def __bool__(self):
     def __repr__(self):
         return f'<{self.__class__.__name__} (length={len(self)}) at {id(self)}>'
 
-    def __add__(self, other: 'Document'):
+    def __add__(self, other: Union['Document', Sequence['Document']]):
         v = type(self)()
-        for doc in self:
-            v.append(doc)
-        for doc in other:
-            v.append(doc)
+        v.extend(self)
+        v.extend(other)
```

Comment:
why not self.extend(other) and return self?

Reply:
`__add__`
a  + b

not 

`__iadd__`
a+=b